gemoji/src/main/kotlin/io/sweers/catchup/gemoji/GemojiDatabase.kt

```diff
@@ -22,7 +22,7 @@ import android.arch.persistence.room.Room
 import android.arch.persistence.room.RoomDatabase
 import android.content.Context
 
-fun GemojiDatabase(context: Context, name: String): GemojiDatabase {
+fun createGemojiDatabase(context: Context, name: String): GemojiDatabase {
```

Comment:
I think it makes sense to remove this function and just inline it inside of `provideGemojiDatabase`, waiting until the dagger module gets moved.